process/sync/metablock_test.go

```diff
@@ -2552,3 +2552,56 @@ func TestMetaBootstrap_ApplyNotarizedBlockShouldWork(t *testing.T) {
 
 	assert.Nil(t, err)
 }
+
+func TestMetaBootstrap_SetStatusHandlerNilHandlerShouldErr(t *testing.T) {
+	t.Parallel()
+
+	pools := &mock.MetaPoolsHolderStub{}
+	pools.MetaChainBlocksCalled = func() storage.Cacher {
+		sds := &mock.CacherStub{}
+
+		sds.HasOrAddCalled = func(key []byte, value interface{}) (ok, evicted bool) {
+			assert.Fail(t, "should have not reached this point")
```

Comment:
why error? are we testing that in a certain condition  HasOrAdd is not called?

Reply:
removed